### DISCUSSION TITLE
water visualization

### DISCUSSION THREAD

**Discussion by Niveusater** [shunnkou/perry-bot#15] [Ideas]
ok so i know we are both bad at coding bUT this could be a good idea so

cups and bottles come in different shapes and sizes. a good way to gauge how much someone is drinking is through mathematical conversions 

so what you can start with is by asking the user "yo how much can your cup hold" and they can give some sort of conversion into ml, liters, or whatever imperial bs unit that we use

yes i still have a functional brain i hope

it would look something like this

x = (ask for how much water their standard cup holds)

fancy math shit 

add it together for a weekly visualization of how much water you drink and you can standardize multiple stuff!! like bottles and different cups

ez water visualization

**Comment by shunnkou** [comment]
One question, does this mean that someone has to specify which cup they want to log with when using the `water` command? Wouldn't that get a bit cumbersome or annoying to remember?

**Reply by Niveusater** [reply]
it will get cumbersome, i agree, so there should be a way for you to maybe set that as "the standard" cup or bottle or something? like it will log it as that unless told otherwise

**Reply by shunnkou** [reply]
That sounds good. We can also add an option that responds with a table of the cups you have and their capacity.

Also, need to make a `config.yml` file
